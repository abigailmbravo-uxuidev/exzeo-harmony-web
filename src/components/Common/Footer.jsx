import React from 'react';

const Footer = () =>
  <footer>
    <div role="banner">
      <small>
        &copy;2017 TypTap Management Company. All rights reserved.
      </small>
    </div>
  </footer>;

export default Footer;
