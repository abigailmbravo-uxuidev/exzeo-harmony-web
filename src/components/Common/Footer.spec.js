import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import Footer from './Footer';

const middlewares = [];
const mockStore = configureStore(middlewares);
it('renders without crashing', () => {
  const props = {
    auth: {
      userProfile: {}
    }
  };
  const initialState = {
    cg: {
      bb: {
        data: {
          modelInstanceId: '123',
          model: {},
          uiQuestions: []
        }
      }
    },
    appState: {
      data: { showScheduleDateModal: true },
      modelName: 'bb'
    }
  };
  const store = mockStore(initialState);
  const wrapper = shallow(<Footer store={store} {...props} />);
  expect(wrapper);
});
