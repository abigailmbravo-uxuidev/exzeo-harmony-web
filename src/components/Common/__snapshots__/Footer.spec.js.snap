// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<footer>
  <div
    role="banner"
  >
    <small>
      Â©2017 TypTap Management Company. All rights reserved.
    </small>
  </div>
</footer>
`;
